arduino/sketch/sketch.go

```diff
@@ -93,6 +94,15 @@ func New(path *paths.Path) (*Sketch, error) {
 		Metadata:         new(Metadata),
```

Comment:
```suggestion
		Metadata:         new(Metadata),
		Project:          new(Project),
```

Otherwise `github.com/arduino/arduino-cli/arduino/sketch.Sketch.GetProfile` will panic under the following conditions:

- The command uses `--profile`
- The sketch does not contain a project file